debezium-core/src/main/java/io/debezium/relational/history/ConnectTableChangeSerializer.java

```diff
@@ -45,6 +46,9 @@ public class ConnectTableChangeSerializer implements TableChanges.TableChangesSe
     public static final String AUTO_INCREMENTED_KEY = "autoIncremented";
     public static final String GENERATED_KEY = "generated";
     public static final String COMMENT_KEY = "comment";
+    public static final String HAS_DEFAULT_VALUE = "hasDefaultValue";
```

Comment:
Hi, I wonder if we need this one. In case of default value not being present `defaultValueExpression` would just be omitted. If present but `null` then it will be set to `null`.